examples/plot_metrics_with_error.py

```diff
@@ -0,0 +1,162 @@
+# %%
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+================================
+Plotting Metrics with Error Bars
+================================
+"""
+# %%
+# Load and preprocess the data set
+# --------------------------------
+# We download the data set using `fetch_openml` function in `sklearn.datasets`.
+# The original Adult data set can be found at https://archive.ics.uci.edu/ml/datasets/Adult
+# We start by importing the various modules we're going to use:
+#
+import os
+import sys
+nb_dir = os.path.split(os.getcwd())[0]
+if nb_dir not in sys.path:
+    sys.path.append(nb_dir)
+
+import numpy as np
+import pandas as pd
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.metrics import recall_score, accuracy_score, confusion_matrix
+from sklearn.datasets import fetch_openml
+from fairlearn.metrics import MetricFrame
+from fairlearn.experimental import enable_metric_frame_plotting
+
+
+data = fetch_openml(data_id=1590, as_frame=True)
+X = pd.get_dummies(data.data)
+y_true = (data.target == '>50K') * 1
+sex = data.data['sex']
+
+# %%
+# Error Metrics
+# -------------
+# We have many different choices for error metrics. In this notebook we'll just be using a
+# Normal approximation interval (symmetric) and a Wilson score interval (asymmetric).
+#
+# https://en.wikipedia.org/wiki/Binomial_proportion_confidence_interval#Wilson_score_interval
+# https://en.wikipedia.org/wiki/Binomial_proportion_confidence_interval#Normal_approximation_interval
+#
+# We aim to create a 95% confidence interval, so we use a `z_score` of 1.959964
```

Comment:
Reading the linked wikipedia page, it appears that this is due to  '1.959964' meaning the number of standard deviations.... if so, then the text needs to say this explicitly. In general, I would also wonder if '1.959964' is _meaningfully_ different from '2' in this context.